spec/views/game_invitations/create.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "game_invitations/create.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
We don't need specs for views. Can you delete this file and see if there's a way to disable the generation of view specs?